libopenage/screenshot.cpp

```diff
@@ -6,20 +6,24 @@
 #include <epoxy/gl.h>
 #include <math.h>
 #include <memory>
+#include <png.h>
+#include <setjmp.h>
 #include <stdlib.h>
-#include <SDL2/SDL.h>
-#include <SDL2/SDL_image.h>
+#include <stdio.h>
+#include <string.h>
```

Comment:
in c++ it's better to use [`csetjmp`](http://en.cppreference.com/w/cpp/utility/program/setjmp), `cstdio` and `cstring`, because that namespaces things with `std::`. Doesn't matter much, though.